snippet_sources.json

```diff
@@ -109,5 +109,17 @@
   {
     "src": "https://raw.githubusercontent.com/hyperledger/iroha-javascript/iroha2/packages/docs-recipes/src/9.blocks-stream.ts",
     "filename": "js-sdk-9-blocks-stream.ts"
+  },
+  {
+    "src": "https://raw.githubusercontent.com/hyperledger/iroha-java/iroha2-dev/modules/test-tools/src/main/kotlin/jp/co/soramitsu/iroha2/testengine/IrohaConfig.kt",
+    "filename": "IrohaConfig.kt"
```

Comment:
You can either omit `filename` at all (as it is the same as in `src`), or specify it as `IrohaConfig.kotlin`, thus you will not need to write `{kotlin}` in snippet import syntax.